spec/griddler/email_spec.rb

```diff
@@ -432,6 +432,15 @@ def email_with_params(params)
     expect(headers[header_name]).to eq header_value
   end
 
+  it 'handles a hash being submitted' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.